apps/webClient/src/components/View/index.tsx

```diff
@@ -99,12 +108,35 @@ const View: React.FC<Props> = ({ children }) => {
         <Box mt={deriveLogoBoxMarginTop()} mb={12}>
           <Grid container item justify="center">
             <Link to="/">
-              <Logo height={deriveLogoHeight()} />
+              <Logo height={deriveLogoHeight()} svgColor="white" />
```

Comment:
nitpick (that I didn't notice earlier) (no need to change)

just use `color` as a prop, no need to specify that its `svg`